third_party/download_source.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+while read repo
+do
+  echo "Fetching source code for repo: github.com/$repo"
+  curl --create-dirs -o "$OUTPUT_FOLDER$repo.source.zip" -L "https://github.com/$repo/archive/master.zip"
+done < "/dev/stdin"
```

Comment:
if we move `set -e` to the "hashbang" (first line if this file), then `< "/dev/stdin"` won't be needed.

Reply:
Thanks! Today I Learned.